### ISSUE TITLE
Add emergency POIs?

### ISSUE THREAD

**Issue by HMT-HRO-MaMe** [indoorequal/indoorequal.org#57]
Hi,

while I'm not sure, if it's wise + allowed to map emergency equipment of buildings at OSM, I guess it's possible. Rescue plans are obligatory here in Germany for any public building, so you might share this details also online, to teach visitors where they can leave the building, or get help for rescue:
* entrance=emergency
* emergency=defibrillator
* emergency=assembly_point
* emergency=landing_site
* emergency=dry_riser_inlet
* emergency=fire_alarm_box
* emergency=fire_extinguisher
* emergency=fire_hose
* emergency=fire_hydrant
* emergency=life_ring
* emergency=assembly_point

Maybe this should be a seperated layer to avoid visual overload?
There are maps like [openfiremap.de](http://www.openfiremap.de/?zoom=15&lat=54.08671&lon=12.12869&layers=B00000T) or [Wambacher's Emergency](https://wambachers-osm.website/Emergency.html?map=15/54.083701/12.13882&base=OSM&data=EMSVS,EMPHN,EMASP,EMACP,SIR,FST,HYD,EMAST,EMDEF,EMFAK,EMWE,EMLND&menu=ROOT,BASE,OVL,EMG,FIRE,OTHR).

**Comment by francois2metz** [comment]
`entrance=emergency` and `emergency=defibrillator` are already supported. Other values seems to be quite niche, not sure we want to support them.

**Comment by HMT-HRO-MaMe** [comment]
Yeah, it's just an idea that came up with some discussion with collegues

**Comment by francois2metz** [comment]
> I guess assembly_point is some kind of information which is also important to the most visitors and to show them (before) an indident?

I think most assembly point should not be indoor. 

Regarding other emergency values: https://taginfo.indoorequal.org/keys/emergency#values `emergency=phone` and `emergency=fire_extinguisher` seems to be quite used. I'll add support to them.

**Comment by francois2metz** [comment]
Vector tile support added in https://github.com/indoorequal/indoorequal/commit/6d6fa653dc7c03f3b4bca2652da5cc7f3da1f2e5.
